src/templates/article.js

```diff
@@ -200,20 +232,24 @@ const Article = props => {
             <FeedbackContainer />
             <Container>
                 <Icons>
-                    <ContentsMenu
-                        title="Contents"
-                        headingNodes={headingNodes}
-                        height={size.default}
-                        width={size.default}
-                    />
-                    <ShareMenu
-                        title={articleTitle}
-                        url={articleUrl}
-                        height={size.default}
-                        width={size.default}
-                    />
+                    <MenuContainer>
+                        <ContentsMenu
+                            title="Contents"
+                            headingNodes={headingNodes}
+                            height={size.default}
+                            width={size.default}
+                        />
+                        <ShareMenu
+                            title={articleTitle}
+                            url={articleUrl}
+                            height={size.default}
+                            width={size.default}
+                        />
+                    </MenuContainer>
+                    <ArticleRating isTop />
```

Comment:
Did we lose the highlight effect for the stars? They now blend into the background on a hover

![Screen Shot 2021-04-14 at 2 03 54 PM](https://user-images.githubusercontent.com/9064401/114757963-75794080-9d2a-11eb-8511-8f79949e9ce4.png)